src/clippy.c

```diff
@@ -534,6 +534,14 @@ clippy_method_call (GDBusConnection       *connection,
   GVariant *return_value = NULL;
   Clippy *clip = user_data;
   GError *error = NULL;
+  GApplication *app;
+
+  /* Make sure the app is activated in case it was autostarted
+   * through this method call.
```

Comment:
How did this work before the change to export the Clippy interface differently?

Reply:
It worked accidentally because the HackUnlock app was not being spawned with `—gapplication-service`, so it was always activated when spawned.
Another approach if we don’t do this here, is to have the FBE explicitly call Activate() over DBus on the HackUnlock application. But it seemed sensible to ensure that Clippy does this as it requires the UI being constructed.

Reply:
https://github.com/endlessm/hack-toy-apps/pull/25 is where that activation issue was fixed.